src/containers/pages/Home/Home.css

```diff
@@ -1,17 +1,17 @@
-.btn-intervention {
+.btn-home-call-to-action {
```

Comment:
What is the intent behind these changes here?

Reply:
My thoughts were, since i added teams button to the home page, but unlike the buttons that were already there i.e. IRS and FI, I thought it did not fall under intervention category so i changed the classname to be more generic to cover the teams button too.
should i revert this?